### PR TITLE
[pull request template] fix internal links

### PR DESCRIPTION
**Pull request by frzifus**
This pr replaces the relative markdown links with external ones. This pr replaces the relative markdown links with external ones. Also line breaks have been removed. 

You can see the result here:

---

## Old Template

Please provide a brief description of the changes here.

For non-trivial changes, follow the [change proposal
process](../CONTRIBUTING.md#proposing-a-change) and link to the related issue(s)
and/or [OTEP(s)](https://github.com/open-telemetry/oteps), update the
[`CHANGELOG.md`](../CHANGELOG.md), and also be sure to update
[`spec-compliance-matrix.md`](../spec-compliance-matrix.md) if necessary.

Related issues #

Related [OTEP(s)](https://github.com/open-telemetry/oteps) #

---

## New Template

Please provide a brief description of the changes here.

For non-trivial changes, follow the [change proposal process](https://github.com/open-telemetry/opentelemetry-specification/blob/main/CONTRIBUTING.md#proposing-a-change) and link to the related issue(s) and/or [OTEP(s)](https://github.com/open-telemetry/oteps), update the [`CHANGELOG.md`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/CHANGELOG.md), and also be sure to update [`spec-compliance-matrix.md`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/spec-compliance-matrix.md) if necessary.

Related issues #

Related [OTEP(s)](https://github.com/open-telemetry/oteps) #

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -4,11 +4,7 @@ Fixes #
 
 Please provide a brief description of the changes here.
 
-For non-trivial changes, follow the [change proposal
-process](../CONTRIBUTING.md#proposing-a-change) and link to the related issue(s)
-and/or [OTEP(s)](https://github.com/open-telemetry/oteps), update the
-[`CHANGELOG.md`](../CHANGELOG.md), and also be sure to update
-[`spec-compliance-matrix.md`](../spec-compliance-matrix.md) if necessary.
+For non-trivial changes, follow the [change proposal process](https://github.com/open-telemetry/opentelemetry-specification/blob/main/CONTRIBUTING.md#proposing-a-change) and link to the related issue(s) and/or [OTEP(s)](https://github.com/open-telemetry/oteps), update the [`CHANGELOG.md`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/CHANGELOG.md), and also be sure to update [`spec-compliance-matrix.md`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/spec-compliance-matrix.md) if necessary.
```
Comment (reyang) [inline .github/PULL_REQUEST_TEMPLATE.md:7]: Would you rewrap this to avoid long line?

**Review by arminru — APPROVED** [review]
Thank you!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (frzifus) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: @reyang The problem is that linebreaks in pr section are not double linebreaks (like usual in markdown). This means that a linebreak also results in a new line.

I hope thats not to confusing. :man_facepalming:

Comment (reyang) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: I see, thanks for the explanation!

**Review by reyang — APPROVED** [review]

**Pull request merged**